x-pack/platform/plugins/shared/ml/server/plugin.ts

```diff
@@ -145,7 +146,7 @@ export class MlServerPlugin
       },
       alerting: Object.values(ML_ALERT_TYPES).map((ruleTypeId) => ({
         ruleTypeId,
-        consumers: [PLUGIN_ID, ALERTING_FEATURE_ID],
+        consumers: [PLUGIN_ID, ALERTING_FEATURE_ID, ...DEPRECATED_ALERTING_CONSUMERS],
```

Comment:
I hadn't noticed that this is a duplicate of `alertingFeatures` from `capabilities.ts`.
We should probably just export `alertingFeatures` and use it here.